### PR TITLE
HPCC-15302 Edit STD.FIle.FileExists topic

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@jakesmith Please review

### CODE REVIEW

**Comment by richardkchapman** [comment]
@jakesmith Please review

File: docs/ECLStandardLibraryReference/SLR-Mods/SuperFileExists.xml [hunk 1]
```diff
@@ -40,12 +40,14 @@
   TRUE if the specified <emphasis>filename</emphasis> is present in the
   Distributed File Utility (DFU) and is a SuperFile. It returns FALSE if the
   <emphasis>filename</emphasis> does exist in the DFU but is not a SuperFile
-  (i.e. is a normal DATASET—use the STD.File.FileExists function to detect
+  (i.e., is a normal DATASET—use the STD.File.FileExists function to detect
```
Comment (jakesmith) [inline docs/ECLStandardLibraryReference/SLR-Mods/SuperFileExists.xml]: Was that new comma intentional? Could also do with rewording/simplifying this sentence I think.
```
(Use Std.File.FileExists to check presence of a standard file)
```
..perhaps..

>..does exist in the DFU

would read better as:

.. does exist in DFU

**Comment by JamesDeFabia** [comment]
@jakesmith Edits made, please have another look

**Comment by jakesmith** [comment]
@richardkchapman - good to merge.

**Pull request merged**